### PR TITLE
Service worker mindset missing images

### LINKED ISSUE #8712: Few broken images in an article
**Describe the bug**
Few broken images in the article 'Service worker mindset'.

**To Reproduce**
1. Go to ['Service worker mindest'](https://web.dev/service-worker-mindset/)

**Expected behavior**
All images in the article should load.

### PR DESCRIPTION
**Pull request by AaronForinton**
Fixes #8712

### CODE REVIEW

File: src/site/content/en/blog/service-worker-mindset/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The service worker layer has its **own lifecycle** independent of the browser ta
 
 In the [Service Workies](https://serviceworkies.com) game we cover the many details of the service worker lifecycle and give you a ton of practice working with it.
 
-{% Img src="image/admin/mOyHcAJW55YiI12VW05a.gif", alt="a new service worker replacing an old one", width="800", height="460" %}
+{% Video src="video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/MhBc1XtdrEGOQn8iXtKZ.mp4", alt="a new service worker replacing an old one", width="800", height="460" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/service-worker-mindset/index.md:44]:
```suggestion
{% Video src="video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/MhBc1XtdrEGOQn8iXtKZ.mp4",
autoplay="true",
    controls="true", 
    loop="true",
    muted="true",
     alt="a new service worker replacing an old one", width="800", height="460" %}
```

File: src/site/content/en/blog/service-worker-mindset/index.md [hunk 2]
```diff
@@ -74,7 +79,7 @@ An active service worker goes on living even after a user leaves your site or cl
 
 In [Service Workies](https://serviceworkies.com) we visualize this concept with Kolohe (a friendly service worker) intercepting and handling requests.
 
-{% Img src="image/admin/YcXrupDTu2m7nlRU4FAz.gif", alt="service worker intercepting http requests", width="800", height="605" %}
+{% Video src="video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/BeJWEOe4TPoMzdBTFVWI.mp4", alt="service worker intercepting http requests", width="800", height="605" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/service-worker-mindset/index.md:82]:
```suggestion
{% Video src="video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/BeJWEOe4TPoMzdBTFVWI.mp4", 
autoplay="true",
controls="true", 
loop="true",
muted="true",
alt="service worker intercepting http requests", width="800", height="605" %}
```

File: src/site/content/en/blog/service-worker-mindset/index.md [hunk 3]
```diff
@@ -203,4 +208,4 @@ Getting into the right mindset while thinking about service workers will help yo
 
 If you want to understand all this by [playing a game](https://gedd.ski/post/mastery-through-play/), then you're in luck! Go play [Service Workies](https://serviceworkies.com) where you'll learn the ways of the service worker in order to slay the offline beasts.
 
-{% Img src="image/admin/HN8hJsK415482mqXmiaI.gif", alt="preview of the Service Workies game", width="800", height="519" %}
+{% Video src="video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/ammLWAa5Kc13D5CgstvK.mp4", alt="preview of the Service Workies game", width="800", height="519" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/service-worker-mindset/index.md:211]:
```suggestion
{% Video src="video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/ammLWAa5Kc13D5CgstvK.mp4",
autoplay="true",
controls="true", 
loop="true",
muted="true",
alt="preview of the Service Workies game", width="800", height="519" %}
```

**Comment by AaronForinton** [comment]
Thank you @rachelandrew - was thinking something along these lines must have gone wrong for the MP4s to show as static images in the preview. Is it good to be merged now?

**Review by rachelandrew — APPROVED** [review]

**Comment by rachelandrew** [comment]
@AaronForinton yep, I had just left it there so remember to show you the extra parameters.

**Pull request merged**